src/main/java/org/openstreetmap/atlas/tags/MinHeightTag.java

```diff
@@ -0,0 +1,17 @@
+package org.openstreetmap.atlas.tags;
+
+import org.openstreetmap.atlas.tags.annotations.Tag;
+import org.openstreetmap.atlas.tags.annotations.Tag.Validation;
+import org.openstreetmap.atlas.tags.annotations.TagKey;
+
+/**
+ * OSM min_height tag: https://taginfo.openstreetmap.org/keys/min_height#values
```

Comment:
Nit: the values link can be removed, it's already available in the taginfo parameter below :)

Reply:
nit: Maybe a href for the link?

Reply:
Removed the ink, keeping the one in the tag info parameter.